oom-frontend/src/app/shared/modules/user/user.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { UserFinderComponent } from './user-finder/user-finder.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Out-Of-Memory-Team/out-of-memory/pullRequest?prid=7533382)
```suggestion
import { UserFinderComponent } from "./user-finder/user-finder.component";
```